code/index.html

```diff
@@ -0,0 +1,159 @@
+        <nav>
+          <ul>
+            <li><a href="#">The Story</a></li>
+            <li><a href="#">Products</a></li>
+            <li><a href="#">Follow Us</a></li>
+            <li><a href="#">Contact</a></li>
+            <li><a href="#">Cart</a></li>
+          </ul>
+        </nav>
+      </div> <!-- container -->
+    </header>
+    <section id="main">
+      <div class="container">
+
+        <div id="landing">
+          <h1 class="logo">BEACHED WHALE</h1>
+          <!--<img class="whaleLogo" src="images/whaleSample.png" alt="Company Logo">-->
+          <h2 class="subLogo">ADVENTURE APPAREL &amp KNICKKNACKS</h2>
+        </div> <!-- #landing-->
+
+        <div id="story">
+          <article class="story_text">
+						<h2><a href="#">As the Story Goes.</a></h2>
+						<img src="url" alt="history pic 1">
+						<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Fusce varius augue nec metus viverra interdum. Sed in gravida lectus. Pellentesque habitant morbi tristique senectus et netu et malesuada fames ac turpis egestas. Mauris ut vulputate nisl.  Maecenas quis auctor dui. Ut porta aliquam lorem, at volutpat orci commodo a. Suspendisse nunc orci, porttitor ac sapien a, rutrum porttitor odio. Phasellus faucibus tristique orci eget dictum. Proin tincidunt vehicula leo sed mattis. Nunc faucibus augue a pellentesque tristique. Proin suscipit in dui ut convallis.</p>
+            <img src="url" alt="history pic 2">
+
+						<p class="readmore"><a href="#">Read more &gt;</a></p>
+						<p class="hide" id="show-this-on-click">Maecenas quis auctor dui. Ut porta aliquam lorem, at volutpat orci commodo a. Suspendisse nunc orci, porttitor ac sapien a, rutrum porttitor odio. Phasellus faucibus tristique orci eget dictum. Proin tincidunt vehicula leo sed mattis. Nunc faucibus augue a pellentesque tristique. Proin suscipit in dui ut convallis.</p>
+						<p class="readless hide"><a href="#">Read less &gt;</a></p>
+					</article>
+        </div><!--#story-->
+        <div id="products">
+          <div class="eight_colums">
+            <div class="one-forth column alpha thumbnail"><img src="http://placehold.it/150x150">
+              <a href="url" title="">
+                <h4>sticker 1</h4>
+                <p>$price</p>
+              </a>
+              <a class="button brown" href="#">Add To Cart</a>
+            </div>
+            <div class="one-forth column thumbnail"><img src="http://placehold.it/150x150">
+              <a href="url" title="">
+                <h4>sticker 2</h4>
+                <p>$price</p>
+              </a>
+            </div>
+            <div class="one-forth column thumbnail"><img src="http://placehold.it/150x150">
+              <a href="url" title="">
+                <h4>sticker 3</h4>
+                <p>$price</p>
+              </a>
+            </div>
+            <div class="one-forth column omega thumbnail"><img src="http://placehold.it/150x150">
+              <a href="url" title="">
```

Comment:
Generally the placeholder for an anchor tag without a value is the `#`. When clicked it will keep you on the same page, and is an indication to you and other developers that this needs to get filled in.